core/conn/jdbc_type2/src/main/java/org/apache/trafodion/jdbc/t2/SQLMXPreparedStatement.java

```diff
@@ -2772,7 +2772,7 @@ private void setLong(int parameterIndex) throws SQLException {
 	/**
 	 * Sets the designated parameter to SQL <tt>NULL</tt>.
 	 * <p>
-	 * <B>Note:</B> You must specify the parameters SQL type (although HP SQL/MX
+	 * <B>Note:</B> You must specify the parameters SQL type (although Trafodion SQL/MX
```

Comment:
Actually, SQL/MX is a reference to the predecessor HP product as well. If you wish, consider changing "Trafodion SQL/MX" to just "Trafodion"